tool/src/c/types.rs

```diff
@@ -119,12 +119,15 @@ mod tests {
         test_file! {
             #[diplomat::bridge]
             mod ffi {
+                #[diplomat::opaque]
+                struct MyOpaqueStruct(UnknownType);
+
                 struct MyStruct {
-                    a: Box<MyStruct>,
+                    a: Box<MyOpaqueStruct>,
```

Comment:
nit: I think this needs to be `& MyOpaqueStruct` and we can later enforce that non opaque structs are fully Copy

Reply:
Good point, I changed that